app/assets/javascripts/admin/addon/templates/customize-themes-show.gjs

```diff
@@ -418,6 +423,31 @@ export default RouteTemplate(
             </section>
           {{/if}}
 
+          {{#if @controller.siteSettings.use_overhauled_theme_color_palette}}
+            {{#unless @controller.model.component}}
+              <section class="form-horizontal theme settings control-unit">
+                <ColorPaletteEditor
+                  @colors={{@controller.model.colorPalette.colors}}
+                  @onLightColorChange={{@controller.onLightColorChange}}
+                  @onDarkColorChange={{@controller.onDarkColorChange}}
+                  @hideRevertButton={{true}}
+                />
+                {{#if @controller.hasChangedColors}}
+                  <DButton
+                    class="admin-theme__save-palette-changes btn-primary"
+                    @icon="check"
+                    @action={{@controller.saveColorChanges}}
+                  />
```

Comment:
I think it will be useful to add a disable state for this button since it involves an AJAX call.